# ui directories
patient_info_screen_dir = "resources/assets/ui/patient_info.ui"
patient_record_screen_dir = "resources/assets/ui/patient_records.ui"

database_directory = "database/database.db"

image_extensions = "(*jpg; *jpeg; *png; *bmp; *bjpeg; *gif; *webp; *heic)"
video_extensions = "(*mp4; *m4p; *mpeg; *mpg; *mkv; *webm; *gifv; *gif; *mov; *qt; *wmv)"

media_directory_name = "patient_data"

datetime_format = '%d/%m/%Y'

# PyQt5 Widget Names
add_image_button = "add_image_button"
add_video_button = "add_video_button"
actionShow_Patients = "actionShow_Patients"
actionGo_Back = "actionGo_Back"
edit_record_button = "edit_record_button"
add_delete_record_button = "add_delete_record_button"
actionAdd_New_Patient = "actionAdd_New_Patient"
actionQuit = "actionQuit"
patient_info_screen_header = "patient_info_screen_header"
last_visited_date = "last_visited_date"
date = "date"
phone_number = "phone_number"
necessary_images = "necessary_images"
necessary_videos = "necessary_videos"
